<?xml version="1.0" encoding="UTF-8"?>
<testsuites>
    <testsuite name="force.apex" timestamp="2021-04-14T13:45:35+02:00" hostname="https://data-page-19352-dev-ed.cs101.my.salesforce.com" tests="1" failures="1"  errors="0"  time="2.86">
        <properties>
            <property name="outcome" value="Failed"/>
            <property name="testsRan" value="1"/>
            <property name="passing" value="0"/>
            <property name="failing" value="1"/>
            <property name="skipped" value="0"/>
            <property name="passRate" value="0%"/>
            <property name="failRate" value="100%"/>
            <property name="testStartTime" value="Apr 14, 2021 1:45 PM"/>
            <property name="testExecutionTime" value="2.86 s"/>
            <property name="testTotalTime" value="2.86 s"/>
            <property name="commandTime" value="5.87 s"/>
            <property name="hostname" value="https://data-page-19352-dev-ed.cs101.my.salesforce.com"/>
            <property name="orgId" value="00D1X00000034R1UAI"/>
            <property name="username" value="test-in7nqa4fsmgt@example.com"/>
            <property name="testRunId" value="7071X00001XHipG"/>
            <property name="userId" value="0051X000007SSFeQAO"/>
        </properties>
        <testcase name="should_populate_account_chiffreAffaire" classname="TestUpdateAccountBatch" time="2.86">
            <failure message="System.AssertException: Assertion Failed: Expected: 1500, Actual: 0"><![CDATA[Class.TestUpdateAccountBatch.should_populate_account_chiffreAffaire: line 112, column 1]]></failure>
        </testcase>
    </testsuite>
</testsuites>
